src/Datasource/RepositoryInterface.php

```diff
@@ -24,6 +24,7 @@
     /**
      * Returns the table alias or sets a new one
      *
+     * @deprecated 3.4.0 Use setAlias()/getAlias() instead.
```

Comment:
I'm sorry, but I don't know exactly what to do here regarding the doc annotation 😥